src/Framework/FieldsAPI/PaymentGateways.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Give\Framework\FieldsAPI;
+
+class PaymentGateways extends Field
+{
+    public function getNodeType(): string
+    {
+        return 'layout';
```

Comment:
This is usually only set on the base, or abstract, node class and isn't needed here unless there is some specific use case.

Reply:
I did this simply because I wasn't sure how to categorize the `Gateways` node. In the form rendering engine I made it a layout, but perhaps that's not accurate. It technically is a radio field. So perhaps it's a Field?

Reply:
I would say Field, which is already set by the base Field class.

Reply:
Thanks for bringing this up, @kjohnson! I took another stab at this and I'm much happier with it. I've cleaned up quite a bit and now this actually just acts like a fancy radio field. Tentatively resolved in 2566434